drupal/web/modules/custom/stuartc_tests/tests/src/Kernel/JsonApiRouteTest.php

```
@@ -0,0 +1,100 @@
+    $entity_type_manager = $this->container->get('entity_type.manager');
+
+    $node_type = $entity_type_manager->getDefinition('node');
+    $this->assertNotNull($node_type);
+    $this->assertEquals('node', $node_type->id());
+  }
+
+  /**
+   * Tests that taxonomy term entity type has JSON:API resource type.
+   */
+  public function testTaxonomyEntityTypeHasJsonApiResourceType() {
+    $resource_type_repository = $this->container->get('jsonapi.resource_type.repository');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$resource_type_repository'. (undefined)

(UnusedLocalVariable)